examples/spark/spark-gluster/spark-master.yaml

```diff
@@ -0,0 +1,26 @@
+kind: Pod
```

Comment:
The master shouldn't be run as just a pod.
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/pods.md#durability-of-pods-or-lack-thereof